### DISCUSSION TITLE
Discuss filetype options for storing & loading text & songs as well as matching CPU cursor speed to audio playback

### DISCUSSION THREAD

**Discussion by mxd1760** [mxd1760/song-typing#29] [Ideas]

**Comment by mxd1760** [comment]
While a plain text file might be the easiest to implement for direct text storage, I'm unsure if there might be an existing file type that stores text along with the music which might be better for our use case.

There's also the consideration of what would be easiest for users to input. having a repository might be convenient but at this point, it is a stretch goal, the easier it is for people to get it working on their own.

**Comment by mxd1760** [comment]
Doing a bit of looking I found [this](https://en.wikipedia.org/wiki/LRC_(file_format)). It might be better in several ways but it also makes more work in others. especially considering we'd probably have to fetch data and format it this way as I don't know if any of the LRC repositories on Git Hub will work well if we're starting from user-submitted mp3s or links.

**Comment by Jayman2000** [comment]
Here’s a comment that was supposed to go in this thread: <https://github.com/mxd1760/song-typing/discussions/27#discussioncomment-10993934>
